doc/man1/openssl-fipsinstall.pod.in

```diff
@@ -191,9 +191,10 @@ See RFC 7627 for information related to EMS.
 Configure the module to not allow truncated digests to be used with Hash and
 HMAC DRBGs.  See FIPS 140-3 IG D.R for details.
 
-=item B<-no_dsa_sign_check>
+=item B<-dsa_sign_check>
```

Comment:
Perhaps it should be named `dsa_sign_disabled` so it is obvious what is the effect?